http-client/src/main/java/io/airlift/http/client/Request.java

```diff
@@ -102,8 +103,8 @@ public String getMethod()
     public String getHeader(String name)
```

Comment:
can you add a test that will check whether we can get header regardless of the casing?

Reply:
```
    @Test
    public void testHeaderCaseSensitiveness()
    {
        Request request = Request.builder()
                .setMethod("GET")
                .setUri(URI.create("http://example.com"))
                .setHeader("mixedHeader", "mixed")
                .setHeader("alllowercase", "lower")
                .setHeader("ALLUPPERCASE", "upper")
                .build();

        assertThat(request.getHeader("mixedHeader")).contains("mixed");
        assertThat(request.getHeader("mixedheader")).contains("mixed");
        assertThat(request.getHeader("MIXEDHEADER")).contains("mixed");

        assertThat(request.getHeader("alllowercase")).contains("lower");
        assertThat(request.getHeader("ALLLOWERCASE")).contains("lower");
        assertThat(request.getHeader("AllLowercase")).contains("lower");

        assertThat(request.getHeader("alluppercase")).contains("upper");
        assertThat(request.getHeader("ALLUPPERCASE")).contains("upper");
        assertThat(request.getHeader("AllUppercase")).contains("upper");
    }
```